static/js/protein/data_processing_utils.ts

```diff
@@ -338,6 +343,7 @@ export function getVarGeneAssoc(data: GraphNodes): ProteinVarType[] {
       continue;
     }
     for (const node of neighbour.nodes) {
+      let assocID = null;
```

Comment:
don't shorten associationId here because assoc is not very clear

Reply:
Done!